connect/runtime/src/main/java/org/apache/kafka/connect/util/KafkaBasedLog.java

```diff
@@ -255,20 +255,15 @@ private void readToLogEnd() {
         }
 
         Map<TopicPartition, Long> endOffsets = new HashMap<>();
-        try {
-            poll(0);
```

Comment:
@ewencp 
though the timeout=0, KafkaConsumer returns immediately with any records that are already available;
if this poll(0) read out message of "task config commit", the consumed callback;